float64[] q_start
float64[] q_target
float64   v_max
float64   a_max
---
# Result
bool success
---
# Feedback
float32 percent_complete
